package.json

```
@@ -2,10 +2,10 @@
   "name": "@janiscommerce/request",
   "version": "1.0.0",
   "description": "Simple and ligthweight module to handle external requests",
-  "main": "lib/index.js",
+  "main": "lib/request.js",
   "scripts": {
     "test": "export TEST_ENV=true; mocha --exit -R nyan --recursive tests/",
-    "test-ci": "nyc --reporter=html --reporter=text mocha --recursive tests/",
+    "test-ci": "nyc --reporter=html --reporter=text mocha --exit --recursive tests/",
```

:warning: Si te fijas el workflow del Action de Coverage esta fallando:

https://github.com/janis-commerce/request/pull/1/checks?check_run_id=1724429088

Eso es porque este script esta mal.

Asi debería ser

```suggestion
    "test-ci": "nyc --reporter=lcov --reporter=text-summary mocha --exit --recursive tests/",
```